nessus-agent/elx/configure.sls

```diff
@@ -0,0 +1,68 @@
+# This salt state installs the Nessus Agent using the provided
+# URL to the rpm source file.  Once installed, the agent will be
+# configured and linked to the Nessus server using the provided
+# parameters (Nessus 64 hexadecimal-digit key, Nessus server FQDN and
+# port, Nessus Agent Group Name).  The final step is to start tHe
+# Nessus Agent service.
+#################################################################
+{%- from tpldir ~ '/map.jinja' import nessus with context %}
+{%- set chkFile = '/etc/tenable_tag' %}
+{%- set staleDbs = salt.file.find('/opt/nessus_agent/var/nessus/', maxdepth='1', type='f', name='*.db') %}
+
+{%- if (nessus.nessus_key and nessus.nessus_server and nessus.nessus_port and nessus.nessus_groups) %}
```

Comment:
I believe it is safe to remove this outer if statement now. This are _required_ values to configure the agent. If someone calls the `configure` sls, they _must_ provide these values.